src/components/userAuth/signInMail/SignInMail.js

```diff
@@ -127,7 +129,8 @@ const SignInMail = () => {
         variant="contained"
         disableElevation
         disableRipple
-        className={classes.loginBtn}
+        disabled={!inputEmail || !inputPassword ? true : false}
```

Comment:
smth wierd, maybe you need
`!!inputPassword`

Reply:
The button must be disabled if no password or email is entered. Both fields (email and password) must be filled

Reply:
Thanks, understood. Really unnecessary